### ISSUE TITLE
Whitening and second order optimization both destroy information about the dataset, and can make generalization impossible 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#58]
# TL;DR
Whitening eliminates per-category data correlations and makes it impossible to distinguish between noise and signal; second-order optimization can have the same effect as whitening, and both have a negative impact on generalization performance. However, the training itself becomes faster, and with appropriate regularization, second-order optimization can produce good generalization performance, which may be good for situations where computational resources are a bottleneck.
<img width="602" alt="ZCA whitened" src="https://user-images.githubusercontent.com/24524018/91716189-136c3080-ebca-11ea-9184-5633b38bcd50.png">

# Why it matters:

# Paper URL
https://arxiv.org/abs/2008.07545

# Submission Dates(yyyy/mm/dd)
2020/08/17

# Authors and institutions
Neha S. Wadia, Daniel Duckworth, Samuel S. Schoenholz, Ethan Dyer, Jascha Sohl-Dickstein
- University of California, Berkeley 
- Google Brain

# Methods

# Results

# Comments
